packages/main/src/components/save-password/SavePassword.jsx

```diff
@@ -29,6 +30,10 @@ const SavePassword = () => {
       if (response.status === 200) {
         return response.statusText;
       }
+    },
+    endpointUrl: FORM_ENDPOINT
+  });
+  const { t } = useTranslation();
```

Comment:
What's happening here? I'm seeing this is being declared inside a useffect. Why?

Reply:
These codes were not even there at all at the time I pulled and worked on these translations, seeing it here now is just surprising to me and the line of function for translating initially is now conflicting with the new codes for those lines.

I have helped sort the code out though and committed.. Thanks @JulianaSau